Tutorials/03-Wifi_Getting_Started_Guide.md

```diff
@@ -1,161 +1,153 @@
 
-The battery connector on the back can accept 3V to 6V DC power input. The pushbutton is a reset button. For more information on the Wifi Hardware, visit the [OpenBCI Wifi](http://docs.openbci.com/Hardware/11-Wifi) page in the Hardware section. To update the shield to the latest firmware please visit the [OpenBCI Wifi Programming Tutorial](http://docs.openbci.com/Hardware/11-Wifi_Programming_Tutorial) also in the Hardware section.
+The WiFi Shield must be on the same wireless network as your computer, smart phone, or other internet connected device you wish to talk to the device with. When the WiFi Shield can't join a known network, the Shield turns into a WiFi hotspot, and will have a name such as "OpenBCI-A4AD" where the last four digits are hexadecimal and are unique to your WiFi shield. Use any wireless device to connect to launch a captive touch portal from the the WiFi Shield hotspot.
 
+The WiFi shield does not work for Enterprise level security. Use your cellphone as a hot spot or set up your own wifi network. Push The World is planning a WiFi direct connection as another alternative for researchers and those on enterprise networks.
 
-#### 2. OpenBCI Cyton Board
+### Example
 
-![Cyton](../assets/images/wifi_cyton.jpg)
+A WiFi network is called _MeerketManor_, it's password protected, and the iPhone is connected to _MeerketManor_.
 
-Please be sure to have at least v3.x.x Cyton firmware running on your board! If you have firmware v1.0.0 please see the [Upgrade Guide](https://github.com/OpenBCI/OpenBCI_32bit_Library/blob/dev-3.0.0/UPGRADE_GUIDE.md) because you need to upgrade your [bluetooth radio firmware](https://github.com/OpenBCI/OpenBCI_Radios) because the new 3.0.0 code runs of the stability introduced in 2.0.0.
+![iPhone Connected to MeerketManor](../assets/images/wifi_join_network_1.PNG)
 
-#### 3. Your 6V AA Battery Pack & 4 AA Batteries or LiPo Battery
+There is a known issue where it may take several times to bring up the captive touch portal, Push The World is actively seeking a better solution, in the meantime, be patient and try multiple times to connect to the board. Power cycling the board may useful.
 
-![Battery Connection](../assets/images/wifi_battery_connection.jpg)
+![iPhone Connecting to `OpenBCI-A4AD`](../assets/images/wifi_join_network_2.PNG)
 
-Install 4 AA batteries in your battery pack or charge up that LiPo battery. The Wifi shield has a 4-5x larger power draw then the Bluetooth communication system so we recommend using LiPo to save!
+After a couple seconds a captive touch portal will appear on the computer, phone or tablet. Click _Configure WiFi_.
 
-### Seating the Wifi Shield
+![Captive touch portal first screen](../assets/images/wifi_captive_touch_first_screen.PNG)
 
-Remove power to your Cyton and/or Wifi shield.
+On the next screen, see that _MeerketManor_ is listed as a possible network for the WiFi Shield to join. Select _MeerketManor_ and enter the password for the network and press _save_.
```

Comment:
make a note and add a picture to say which blue LED will light up to indicate that the board is connected